packages/flutter/test/widgets/widget_inspector_test.dart

```diff
@@ -498,4 +498,157 @@ void main() {
       expect(service.toObject(propertyJson['objectId']), const isInstanceOf<DiagnosticsNode>());
     }
   });
+
+  testWidgets('WidgetInspectorService creationLocation', (WidgetTester tester) async {
+    final WidgetInspectorService service = WidgetInspectorService.instance;
+    if (!service.isWidgetCreationTracked()) {
+      print('Skipping WidgetInspectorService creationLocation test '
```

Comment:
use the `skip` argument to `testWidgets` to do this.

Reply:
nice! done.